dotCMS/src/main/java/com/dotcms/rendering/velocity/services/ContentletLoader.java

```diff
@@ -115,6 +116,9 @@ public InputStream buildVelocity(final Contentlet content, final PageMode mode,
         sb.append("#set($CONTENT_TYPE_ID='").append(contentTypeId).append("' )");
         sb.append("#set($CONTENT_LANGUAGE='").append(content.getLanguageId()).append("' )");
 
+        final Optional<Integer> pageReferences =
```

Comment:
we need to check if it is on the content map, also if it is not on the map, you can only get the references here if the mode is EDIT MODE

Reply:
This feedback has been addressed - thanks